app/views/comments/_form.html.erb

```diff
@@ -100,15 +100,15 @@
 <% end %>
 
 <% if @comment.persisted? %>
-  <script defer>
+  <script>
     var editForm = document.getElementById("edit_comment_<%= @comment&.id %>");
     if ( editForm ) {
       editForm.onsubmit = function (e) {
         e.preventDefault();
         document.getElementsByClassName("comment-form")[0].classList.add("opacity-50");
-        var form = this;
-        var waitingOnCSRF = setInterval(function () {
-          var metaTag = document.querySelector("meta[name='csrf-token']")
+        const form = this;
```

Comment:
@jmau111, apologies if I did not make myself clear. 🙃 What I meant was revert all the `const`s to `var` so that all we have is the removal of the script `defer` attributes.

Reply:
Hum, have you any example of bug ? Do you get something bad in tests or in your browser's console ?